test/view_component/multiple_formats.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class ViewComponentMultipleFormatsTest < ViewComponent::TestCase
+  def test_render_html
+    render_inline(MyComponentMultipleFormat.new, format: :html)
```

Comment:
Wouldn't the format be set by the request? I'm guessing we'd need to add a `with_request_format` test helper.

Reply:
Ok, I started with a skeleton code/pseudo code.

What I'm thinking to do is:

- check if there is a view for `#{component}.#{format}.#{extension}`. if it exists, set it as view.